asset-properties-widget/component/asset-properties-view/asset-properties/asset-properties.component.ts

```diff
@@ -144,9 +152,10 @@ export class AssetPropertiesComponent implements OnChanges {
               : undefined,
           isEdit: false,
           jsonSchema: mo.c8y_JsonSchema,
-            lastUpdated: mo.lastUpdated,
-            isEditable: mo.isEditable,
-            active: properties[key].active as boolean,
+          lastUpdated: mo.lastUpdated,
+          isEditable: mo.isEditable,
+          active: properties[key].active as boolean,
+          isExistingProperty: mo.isExistingProperty
```

Comment:
```suggestion
          isStandardProperty: mo.isStandardProperty
```
Would isStandardProperty better than existing property if you are adding this for properties for device and computed?

Reply:
incorporated.